packages/ListBoxWithTags/src/helpers.js

```diff
@@ -0,0 +1,11 @@
+export const filter = (search, data, id = "label") => {
+  const regex = new RegExp(`${search}`, "gi");
+  const results = [];
+  for (let i = 0, len = data.length; i < len; i++) {
+    if (data[i][id].match(regex) !== null) {
+      results.push(data[i]);
+    }
+  }
+
+  return results;
+};
```

Comment:
I think this is just:

```
export const filter = (pattern, data, id = "label") => {
  const regex = new RegExp(pattern, "gi");  
  return data.filter(datum => datum[id].match(regex) !== null)
};
```

Reply:
yeah in this specific case I used a `for` instead of the `filter` method because theoretically filter it should be slower than a regular `for`, and while 99% of the cases I would be used the `Array.filter` method, in this case, might get call multiple cases in a long list, so didn't add too much effort to me to add this "small" boost of performance even if I haven't been measure it.